jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ImageParameters.java

```diff
@@ -31,47 +29,25 @@
  * <p>{@code image} (required) is the image reference and {@code credHelper} (optional) is the name
  * (after {@code docker-credential} of the credential helper for accessing the {@code image}.
  */
-public class ImageParameters {
-
-  private AuthParameters auth;
-
-  @Nullable private String image;
-  @Nullable private String credHelper;
-
-  @Inject
-  public ImageParameters(ObjectFactory objectFactory, String imageDescriptor) {
-    auth = objectFactory.newInstance(AuthParameters.class, imageDescriptor + ".auth");
-  }
+interface ImageParameters {
 
   @Input
```

Comment:
Is this `@Input` in effect in implementation classes? Should this be on implementation classes? (Not sure.)